mpv.el

```diff
@@ -73,6 +73,21 @@
   :type 'number
   :group 'mpv)
 
+(defcustom mpv-entry-format "%s [%s]"
```

Comment:
This is only used for chapters / entries with offsets, right? Let's maybe rename this to `mpv-entry-with-offset-format`.